### PR TITLE
Adafruit DHT libraries addition

### PR DESCRIPTION
**Pull request by Puneethkumarsn**

### CODE REVIEW

File: libraries/DHT-sensor-library/DHT-unified-sensor/project.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{ "name": "DHT-sensor-library", "author": "Tony DiCola" }
```
Comment (urish) [inline libraries/DHT-sensor-library/DHT-unified-sensor/project.json:1]: author should by "Rob Tillaart"  no?

Comment (Puneethkumarsn) [reply libraries/DHT-sensor-library/DHT-unified-sensor/project.json:1]: image

Comment (urish) [reply libraries/DHT-sensor-library/DHT-unified-sensor/project.json:1]: Oh, it seems like `DHT_Unified_Sensor.ino` here is the wrong one! Can you please look into it?

Comment (Puneethkumarsn) [reply libraries/DHT-sensor-library/DHT-unified-sensor/project.json:1]: My Bad. Fixed now

**Pull request merged**